packages/core/src/tracing/index.js

```diff
@@ -226,6 +237,14 @@ function initInstrumenations(_config) {
       instrumentationModule.init(_config);
     });
 
+    if (customInstrumentations.length > 0) {
+      // @ts-ignore
+      customInstrumentations.forEach(instrumentationPath => {
```

Comment:
The concept of activating & deactivating is missing.
An instrumentation is deactivated by default. Initializing won't mean it will work.